common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/utils/LwM2mTransportUtil.java

```diff
@@ -452,4 +532,15 @@ private static String opaqueToString(byte[] value) {
         return opaque.length() > 1024 ? opaque.substring(0, 1024) : opaque;
     }
 
+    public static LwM2mModel createModelsDefault() {
+        return new StaticModel(ObjectLoader.loadDefault());
+    }
+
+    public static boolean compareAttNameKeyOta(String attrName) {
+        for (OtaPackageKey value : OtaPackageKey.values()) {
```

Comment:
for (OtaPackageKey value : OtaPackageKey.values()) {
          for(OtaPackageType type : OtaPackageType.values(){
                      if (getAttributeKey(type, value).equals(attrName)) return true;
          }
        }